test/unit-tests/org/listing-usersTest.js

```diff
@@ -0,0 +1,152 @@
+const server = require('../../../test-utils/index')
+const chai = require('chai')
+const expect = chai.expect
+chai.use(require('chai-http'))
+
+const orgFixtures = require('./mockObjects.org')
+const orgController = require('../../../src/controller/org.controller/org.controller')
+const orgParams = require('../../../src/controller/org.controller/org.middleware')
+
+const errors = require('../../../src/controller/org.controller/error')
+const error = new errors.OrgControllerError()
+
+describe('Testing the /org/:shortname/users endpoint in Org Controller', () => {
+    before(async () => {
+
+    })
+
+    context('Negative Tests', () => {
+        it('should return 404 becuase org does not exist', (done) => {
+            class NoOrg {
+                async getOrgUUID () {
+                    return null
+                }
+            }
+        
+            class Blank {
+                async aggregate () {
+                    return []
+                }
+            }
+        
+            server.route('/org-does-not-exist/:shortname/users')
+              .get((req, res, next) => {
+                const factory = {
+                  getOrgRepository: () => { return new NoOrg() },
+                  getUserRepository: () => { return new Blank() }
+                }
+                req.ctx.repositories = factory
+                next()
+              }, orgParams.parseGetParams, orgController.USER_ALL)
+
+            // perform the request to the api
```

Comment:
Nitpick: this change set contains 4-5 "perform request" or "assert response" or "execute" style comments, which more or less reiterate what the next line of code says and could be argued to be redundant/unnecessary.

Reply:
This will be taken out later in all unit tests at once